task5/src/main/java/com/clouway/adapter/http/LoginPage.java

```diff
@@ -25,6 +25,7 @@ public LoginPage(SessionRepository sessionRepository) {
 
   protected void doGet(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
     String errorMessage = request.getParameter("errorMsg");
+    sessionRepository.cleanExpired();
```

Comment:
Third place with `cleanExpired()` added in PR for operation history. Do you review the PR before put `need review` ?